filebeat/filebeat.yml

```diff
@@ -1,205 +1,243 @@
+  #backoff: 1s
+
+  # Max backoff defines what the maximum backoff time is. After having backed off multiple times
+  # from checking the files, the waiting time will never exceed max_backoff independent of the
+  # backoff factor. Having it set to 10s means in the worst case a new line can be added to a log
+  # file after having backed off multiple times, it takes a maximum of 10s to read the new line
+  #max_backoff: 10s
+
+  # The backoff factor defines how fast the algorithm backs off. The bigger the backoff factor,
+  # the faster the max_backoff value is reached. If this value is set to 1, no backoff will happen.
+  # The backoff value will be multiplied each time with the backoff_factor until max_backoff is reached
+  #backoff_factor: 2
+
+  # This option closes a file, as soon as the file name changes.
+  # This config option is recommended on windows only. Filebeat keeps the files it's reading open. This can cause
+  # issues when the file is removed, as the file will not be fully removed until also Filebeat closes
+  # the reading. Filebeat closes the file handler after ignore_older. During this time no new file with the
+  # same name can be created. Turning this feature on the other hand can lead to loss of data
+  # on rotate files. It can happen that after file rotation the beginning of the new
+  # file is skipped, as the reading starts at the end. We recommend to leave this option on false
+  # but lower the ignore_older value to release files faster.
+  #force_close_files: false
+
+#----------------------------- Stdin prospector -------------------------------
+# Configuration to use stdin input
+#- input_type: stdin
+
+#========================= Filebeat global options ============================
+  
+# Event count spool threshold - forces network flush if exceeded
+#filebeat.spool_size: 2048
+
+# Enable async publisher pipeline in filebeat (Experimental!)
+#filebeat.publish_async: false
 
-  # Enable async publisher pipeline in filebeat (Experimental!)
-  #publish_async: false
+# Defines how often the spooler is flushed. After idle_timeout the spooler is
+# Flush even though spool_size is not reached.
+#filebeat.idle_timeout: 5s
 
-  # Defines how often the spooler is flushed. After idle_timeout the spooler is
-  # Flush even though spool_size is not reached.
-  #idle_timeout: 5s
+# Name of the registry file. If a relative path is used, it is considered relative to the
+# data path.
+#filebeat.registry_file: registry
 
-  # Name of the registry file. If a relative path is used, it is considered relative to the
-  # data path.
-  #registry_file: registry
+#
+# These config files must have the full filebeat config part inside, but only
+# the prospector part is processed. All global options like spool_size are ignored.
+# The config_dir MUST point to a different directory then where the main filebeat config file is in.
+#filebeat.config_dir:
+#================================ General =====================================
```

Comment:
New line between would help. Probably best to introduce it on top of libbeat.yml to have it in all files always.